launchlib/config_test.go

```diff
@@ -41,9 +44,13 @@ args:
 		ConfigVersion: 1,
 		MainClass:     "mainClass",
 		JavaHome:      "javaHome",
-		Classpath:     []string{"classpath1", "classpath2"},
-		JvmOpts:       []string{"jvmOpt1", "jvmOpt2"},
-		Args:          []string{"arg1", "arg2"}}
+		Env: map[string]string{
+			"SOME_ENV_VAR":  "/etc/profile",
+			"OTHER_ENV_VAR": "/etc/redhat-release",
```

Comment:
your last comment is in reference to asking about adding variable expansion tests to this specific test (since that wouldn't make sense here given variable expansion happens at execution time)